src/screens/Grommet.js

```diff
@@ -343,6 +343,22 @@ const GrommetPage = () => (
       multiple: "string",
       selected: "string",
     },
+    selectMultiple: {
```

Comment:
Should we add this to here as well with an updated description that says "To affect all SelectMultiple instances, use  [messages ](https://v2.grommet.io/grommet#messages)prop on Grommet tag."

https://v2.grommet.io/selectmultiple#messages

Reply:
aw yes I can add that forgot we were going to add in that little note